src/components/select/multi-select/multi-select.test.tsx

```diff
@@ -1,24 +1,45 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
-import MultiSelect from ".";
-import Option from "../option";
+import {
+  fireEvent,
+  act,
+  render,
+  screen,
+  waitFor,
+  within,
+} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import mockDOMRect from "../../../__spec_helper__/mock-dom-rect";
 import Logger from "../../../__internal__/utils/logger";
+import { testStyledSystemMarginRTL } from "../../../__spec_helper__/__internal__/test-utils";
+
+import MultiSelect from ".";
+import { Option } from "..";
+
+import Modal from "../../modal";
+
+beforeEach(() => {
+  // Mock non-zero dimensions for the scrollable container in dropdown list. To ensure react-virtual renders options in the dropdown list correctly.
+  mockDOMRect(40, 100, "select-list-scrollable-container");
+});
 
-test("renders combobox without text overlay", () => {
+afterEach(() => {
+  jest.restoreAllMocks();
+});
+
+test("displays input field", () => {
```

Comment:
comment: I'm not sure we get much value from this test as we can imply it's presence/visibility from the other tests that query/assert against it